### DISCUSSION TITLE
局面解析又一篇，看引擎的理解能力和判断方向

### DISCUSSION THREAD

**Discussion by ATH139** [official-pikafish/Pikafish#81] [General]
局面如下：
3akabr1/9/2n1b3n/p3p1p1p/2p6/PR7/2c1P3P/r2CB1N2/N1R1A4/5KB2 w - - 0 1
**正招：炮六退二**
建议：关闭云库或关闭云库出招模式，以防干扰引擎思考

提醒：_此局面下的引擎测试结果不代表该棋软的综合实力，请勿以点带面！_

题外话：
该盘面，部分棋软可以瞬秒出正招；部分棋软无法找到正招；个别棋软可以找到正招但打分跳水认为己方可能处于被动…以当前的**皮卡鱼917版**为例，在不改变软件设置且多次计算的情况下；1、引擎经过较长时间计算仍无法找到正招的，2、引擎经过一段时间计算可以找到正招但分数波动在1500左右的。不过值得注意的是：不同于其他软件的其他招法以及云库中的备选招法，917版似乎更青睐于“车八退一”这一步棋，后续走棋纵观整体盘面，鱼的这一步倒也显得匠心独运；鹤立鸡群一般，其独到的布局思维和战略眼光让人有耳目一新的感觉，我想这可能也是皮卡鱼能成功俘获大批拥趸的原因之一吧，介于此，将此局面发上来仅供团队测试及棋迷朋友们解玩使用！顺祝大家国庆快乐，平安喜乐！

**Comment by ATH139** [comment]
注意：在经过917版走出“车八退一”后，云库备选招法中已更新出该招法！

**Reply by leonwerth2703** [reply]
Maybe you will like this fen: 3akc1P1/4a1P2/9/2N2R3/9/9/6r2/3AK4/4A4/9 w, 
![Screenshot 2024-08-16 121312](https://github.com/user-attachments/assets/2671fe73-3fb3-45ba-ac44-9303e2d0a74b)
Pikafish can't not solve this fen even just 1 move and win the game, even cloud book also give wrong move, the best move must be X4.3 like this 
![Screenshot 2024-08-16 121632](https://github.com/user-attachments/assets/1c8df509-8f4f-441a-a93b-ef86ef8e5095)

**Comment by hekunzai** [comment]
底层代码应该是不会主动送子的，所以不会优先考虑
